src/xrCore/LocatorAPI_defs.cpp

```diff
@@ -105,6 +105,17 @@ LPCSTR FS_Path::_update(string_path& dest, LPCSTR src) const
     R_ASSERT(src);
     string_path temp;
     xr_strcpy(temp, sizeof(temp), src);
+
+#ifdef XR_PLATFORM_LINUX
+    string_path fullPath;
+    strconcat(sizeof(fullPath), fullPath, m_Path, temp);
+    xr_fs_strlwr(fullPath);
```

Comment:
Don't use `xr_fs` if you already use `#ifdef XR_PLATFORM_LINUX`, just use `xr_`

Reply:
Done